### PR TITLE
Add `options` method

### PR DESCRIPTION
**Pull request by fujimura**
Addresses https://github.com/scotty-web/scotty/pull/143.

### CODE REVIEW

File: src/Test/Hspec/Wai.hs [hunk 1]
```diff
@@ -115,6 +116,10 @@ put path = request methodPut path []
 patch :: ByteString -> LB.ByteString -> WaiSession SResponse
 patch path = request methodPatch path []
 
+-- | Perform a @OPTIONS@ request to the application under test.
```
Comment (sol) [inline src/Test/Hspec/Wai.hs]: ^ type, this should be "Perform an @OPTIONS@ request..."

**Comment by sol** [comment]
Other than the typo, LGTM.

**Comment by fujimura** [comment]
Good point! Thanks.

**Comment by fujimura** [comment]
@sol Let me merge this. I'd like to make it available for https://github.com/scotty-web/scotty/pull/143.

**Pull request merged**